### PR TITLE
Onboarding Tutorial

### PR DESCRIPTION
**Pull request by lazynina**

### CODE REVIEW

**Review by maebeam — APPROVED** [review]
My only concern is users who currently have `MustPurchaseCreatorCoin = true` will not be forced to complete the tutorial

File: routes/user.go [hunk 1]
```diff
@@ -1078,6 +1104,18 @@ func (fes *APIServer) GetSingleProfile(ww http.ResponseWriter, req *http.Request
 		res.IsGraylisted = true
 	}
 
+	if isAdmin, _ := fes.UserAdminStatus(publicKeyBase58Check); isAdmin {
```
Comment (maebeam) [inline routes/user.go:1107]: Why are we checking if the profile we're fetching is an admin? This seems wrong?

Comment (lazynina) [reply routes/user.go:1107]: Basically, we only need the IsFeaturedTutorialUpAndComing and IsFeaturedTutorialWellKnownCreator values if the user is an admin. These values are only used to determine whether to show "Add" or "Remove" from the well-known and up-and-coming creator indexes on a creator's profile. We don't need to fetch user metadata from global state for anything else, so if we're not an admin viewing a profile, it's a little bit faster.

**Comment by lazynina** [comment]
> My only concern is users who currently have `MustPurchaseCreatorCoin = true` will not be forced to complete the tutorial

I thought that as well, but it felt simpler to remove that MustPurchaseCreatorCoin stuff all together. Happy to add it back in if we decide we want to enforce it. Will defer to @diamondhands0 on that though

File: routes/user.go (hunk 1 above)
Comment (lazynina) [reply routes/user.go:1107]: oh I see - yes we need to update this to take in the reader public key. Good catch!

**Review by redpartyhat — APPROVED** [review]

**Review by diamondhands0 — APPROVED** [review]

**Pull request merged**